lib/dialects/abstract/query-generator.js

```diff
@@ -1519,8 +1535,26 @@ const QueryGenerator = {
 
         mainQueryOrder.push(this.quote(t, model, false, '->'));
       }
+    } else if (typeof options.order === 'string') {
+      // split
+      for (let order of options.order.split(',')) {
+        // get options
+        const orderOption = getOrderOption(order);
+        if(orderOption.index > -1){
+          order = order.substring(0, orderOption.index);
+        }
+        order = _.trim(order);
+        let sql = this.quote(order, model, false, '->');
+        if(orderOption.option){
+          sql += ' ' + orderOption.option;
+        }
+        if (subQuery) {
+          subQueryOrder.push(sql);
+        }
+        mainQueryOrder.push(sql);
+      }
```

Comment:
@felixfbecker The string order option is not removed and you can still use `Sequelize.literal()`.  However, we can continue to support strings without breaking existing functionality.  Sure, custom sanitation methods are prone to being spoofed, however the order clause has a simple syntax where we can easily validate content and its arrangement.  So why loose existing functionality and when it could be extended?  If worst comes to worst, the logic being implemented here will quote a malicious or malformed order string and error out.